### PR TITLE
:lipstick: Keyword jump block redesign block redesign #634

### LINKED ISSUE #634: :art:  Keyword Jump Block Redesign: block redesign
## User stories

As a user, I want the "Keyword Jump Block" component in the app to be redesigned to enhance its usability and visual appeal.

---

Quick links: [Summary](#summary) | [Description](#description) | [Template](#template) | [Example](#example) | [Resources](#resources)

---

### Technical Description
The current design is: 
image

It should be changed to:
image

The files can be found at `libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/`. 

The designs are [here](https://www.figma.com/file/ZGHzSstXOYPMFU1EiqjEHD/Untitled?type=design&node-id=1%3A4&mode=design&t=Hf7DxaUUMQ3BWy57-1)

### Acceptance Criteria

1. The block should look exactly like the designs.
2. Use of `rem`, `em` as opposed to `px`.
3. All changes should be isolated to the following  `libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/`.

### PR DESCRIPTION
**Pull request by levy-web**
# Description

Redesigned keyword jump block.

Fixes #634 

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

Screenshot from 2023-11-03 07-09-38
Screenshot from 2023-11-03 07-09-33

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**: (optional)
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by LemmyMwaura — CHANGES_REQUESTED** [review]
Sync the latest changes.

File: libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/src/lib/components/keyword-jump-block/keyword-jump-block.component.html [hunk 1]
```diff
@@ -1,21 +1,7 @@
...
 
-      <div formArrayName="options" fxFlexFill>
-        <div *ngFor="let keyword of keywordItems.controls; let i = index" class="full-width" fxFlexFill>
-          <span [formGroupName]="i" fxFlex>
-            <div class="inputList">
-              <app-option-input-field [jsPlumb]="jsPlumb" [isNotEndpoint]="false" [isReadOnly]="true" [blockFormGroup]="keywordJumpBlock" [formGroupNameInput]="i" class="input"></app-option-input-field>
-            </div>
-          </span>
-        </div>
-      </div>
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/src/lib/components/keyword-jump-block/keyword-jump-block.component.html:12]: Any reason this has been omitted?

File: libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/src/lib/components/keyword-jump-block/keyword-jump-block.component.html [hunk 2]
```diff
@@ -1,21 +1,7 @@
...
+    <app-default-option-field [jsPlumb]="jsPlumb" [blockFormGroup]="keywordJumpBlock"></app-default-option-field>
+  </form>
 
-      <app-default-option-field [jsPlumb]="jsPlumb" [blockFormGroup]="keywordJumpBlock"></app-default-option-field>
-    </form>
-
-    <button (click)="addNewOption()" mat-flat-button class="add-button" fxLayout="row" fxLayoutAlign="center"
-      fxLayoutGap="10px">
-      {{ 'BOTS.ACTIONS.ADD-OPTION' | transloco }}
-    </button>
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/src/lib/components/keyword-jump-block/keyword-jump-block.component.html:20]: Any reason this has been removed?

File: libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/keyword-jump-block.jsplumb.ts [hunk 3]
```diff
@@ -24,7 +24,7 @@ export function _KeywordJumpBlockDecoratePlumb(
     target: true,
     cssClass: 'block_endpoint',
     endpoint: 'Dot',
-    anchor: 'Left',
+    anchor: [0, 0.3 , 0, 1],
```
Comment (LemmyMwaura) [inline libs/features/convs-mgr/stories/blocks/library/main/src/lib/providers/keyword-jump-block.jsplumb.ts:27]: Just leave this as left

File: libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/src/lib/components/keyword-jump-block/keyword-jump-block.component.html (hunk 2 above)
Comment (levy-web) [reply libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/src/lib/components/keyword-jump-block/keyword-jump-block.component.html:20]: its removed from required design

File: libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/src/lib/components/keyword-jump-block/keyword-jump-block.component.html (hunk 1 above)
Comment (levy-web) [reply libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/src/lib/components/keyword-jump-block/keyword-jump-block.component.html:12]: so as to align with required design

**Comment by levy-web** [comment]
@LemmyMwaura i have merged with develop and fixed required changes, kindly review.

**Review by LemmyMwaura — APPROVED** [review]

File: libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/src/lib/components/keyword-jump-block/keyword-jump-block.component.html (hunk 2 above)
Comment (LemmyMwaura) [reply libs/features/convs-mgr/stories/blocks/library/keyword-jump-block/src/lib/components/keyword-jump-block/keyword-jump-block.component.html:20]: mmh, okay. we'll go with this.

**Pull request merged**